src/include/btree.i

```diff
@@ -1281,7 +1281,7 @@ __wt_page_release(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags)
  */
 static inline int
 __wt_page_swap_func(WT_SESSION_IMPL *session, WT_REF *held,
-    WT_REF *want, uint32_t flags
+    WT_REF *want, bool notfound_ok, bool restart_ok, uint32_t flags
```

Comment:
Although not a whole lot cleaner, you could add two more flags instead of adding two boolean args.  `WT_READ_NOTFOUND_OK` and `WT_READ_RESTART_OK`.  There are a few other flags that are defined and used in other places, but not used within `wt_page_in_func` as precedent.  It makes the callers more readable at a glance with:
```
flags |= WT_READ_NOTFOUND_OK;
WT_RET(__wt_page_swap_func(...));
```
rather than having to figure out which arg is which every time.